tests/test0178.d

```diff
@@ -0,0 +1,7 @@
+// test interface variable declaration
```

Comment:
You don't need 2 tests cases for theses. Let's put thing in one test case and extend it.